__tests__/app.test.js

```diff
@@ -36,3 +36,40 @@ describe("/api/categories", () => {
       });
   });
 });
+
+describe.only("/api/reviews/:review_id", () => {
```

Comment:
We must take out the .only from the test to ensure that the previous tests are still passing 🙂